docs/use/affinity.md

```diff
@@ -0,0 +1,170 @@
+|**Topology key**|TopologyKey is label key on nodes that defines the grouping or topology scope (e.g., zone, hostname) where the rules are enforced.| Node affinity rules do not include a topology key. </br></br>Here are some examples of **topologyKey**:</br><ul><li>kubernetes.io/hostname</li><li>topology.kubernetes.io/zone	</li><li>topology.kubernetes.io/region</li><li>custom node labels (e.g., rack)</li></ul>|
+|**key**|A key in pod affinity is the label assigned to pods, used to define scheduling rules. This label helps the Kubernetes scheduler identify pods and place new pods in relation to them based on affinity or anti-affinity rules.|The key that you set will be a label present in other existing pods within the cluster, affecting whether your pods will be scheduled on the same nodes as those pods.</br></br>Here are some examples of **key**:</br><ul><li>app</li><li>security</li><li>environment</li><li>custom labels (e.g., security, web-store)</li></ul>|
+|**Operator**|The operator field specifies how a label's values match a resource, such as a Pod or node. It establishes the logical condition to determine if a resource satisfies the affinity or anti-affinity rule.</br></br>The following are all logical operators you can use in the operator field:</br><ul><li>**In**: Matches label values in a list</li><li>**NotIn**: Matches label values not in a list.</li><li>**Exists**: Matches when a label key exists, regardless of its value.</li><li>**DoesNotExist**: Matches when a label key does not exist.</li></ul>|When using the **In** and **NotIn** operators, you have to provide the values for the key as well.|
+|**Values**|The values are the specific label values that must match the key for the affinity rule to apply.|Here are some examples for **values**:</br><ul><li>s2</li><li>database</li><li>production</li><li>custom values</li></ul>|
+
+## Use cases for affinity
+
+Here are several detailed use cases for affinity that highlight its diverse applications and the numerous benefits it offers across various contexts.
+
+
+=== "Node affinity"
+    ### Regional scheduling
+
+    Let's consider a use case in which workloads are distributed based on performance requirements, fault tolerance, and regional specifications across designated zones or areas.
+
+    You need to run a workload in the `us-west2 region` for latency optimaztion and to meet specific compliance requirements.
+
+    ```sh
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+            operator: In
+            values:
+            - "us-west2"
+    ```
```

Comment:
Shouldn't we show a screenshot configuring this instead?

Reply:
I didnt have the build so instead added this...